src/scenes/ProgressReports/Detail/ProgressReportDrawer/Steps/TeamHighlight/TeamHighlightStep.graphql

```diff
@@ -0,0 +1,35 @@
+mutation CreateProgressReportHighlight($input: ChoosePrompt!) {
+  createProgressReportHighlight(input: $input) {
+    ...ProgressReportItemEdit
+    parent {
+      ... on ProgressReport {
+        id
+        highlights {
+          ...Pagination
+          items {
+            id
+          }
+        }
+      }
+    }
+  }
+}
+
+mutation UpdateProgressReportHighlightResponse(
+  $input: UpdatePromptVariantResponse!
+) {
+  updateProgressReportHighlightResponse(input: $input) {
+    ...ProgressReportItemEdit
+    parent {
+      ... on ProgressReport {
+        id
+        highlights {
+          ...Pagination
+          items {
+            id
+          }
+        }
+      }
+    }
```

Comment:
Actually it does need to update the list of highlights as that is how we are **currently** checking if it needs to display a prompt or the variant responses, just fyi @CarsonF 
https://github.com/SeedCompany/cord-field/pull/1188/files/da7f7d13c4e133a74c6570534aa6a0be89c8a45c#diff-62977201f87c94179cc270364dadd3d3ce0941a77208b06a3a81e558595199a5R12

Reply:
Well what I mean is it doesn't directly update the list. Since the list is normalized, it doesn't matter here.
So
```
highlights: ['PVR:abc', 'PVR:def']
```
Now we modify `PVR:def` by submitting a response, and we receive back the that updated object.
So `PVR:def` is now includes a new/updated response, and that cache is updated. Since the `highlights` list is just referencing that normalized cache key it will get the new data when querying the cache.

Reply:
I am sorry, I just realized it was for the UpdateResponse, not CreateHightlight! This will be removed, yes!